internal/git/cmdgit/git.go

```diff
@@ -46,6 +47,15 @@ func (g *Git) run(cmd *exec.Cmd) (string, error) {
 	return stdout.String(), nil
 }
 
+func logGitExecution(cmd *exec.Cmd, stdout *bytes.Buffer, stderr *bytes.Buffer) {
+	logger := log.WithFields(log.Fields{
+		"cmd":    cmd,
```

Comment:
I'm not a fan of relying on `cmd` being "magically" converted to the list of arguments here. Let's be explicit instead.

```suggestion
		"cmd":    cmd.String(),
```